Anonymize/Type/FixedValueAnonymizer.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * This file is part of the GDPR bundle.
+ *
+ * @category  Bundle
+ * @package   Gdpr
+ * @author    SuperBrave <info@superbrave.nl>
+ * @copyright 2018 SuperBrave <info@superbrave.nl>
+ * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
+ * @link      https://www.superbrave.nl/
+ */
+
+namespace SuperBrave\GdprBundle\Anonymize\Type;
+
+use SuperBrave\GdprBundle\Manipulator\PropertyManipulator;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Email anonymizer class
+ *
+ * @package SuperBrave\GdprBundle\Anonymize\Type
+ */
+class FixedValueAnonymizer implements AnonymizerInterface
```

Comment:
The annotation type is now `fixed` but the class name is still `FixedValueAnonymizer` shouldn't we rename it to `FixedAnonymizer`?

Reply:
@TomSchillemans